lib/BulkRenewalDialog/BulkRenewedLoansList.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import Popover from '@folio/stripes-components/lib/Popover';
+import Icon from '@folio/stripes-components/lib/Icon';
+
+import MultiColumnList from '@folio/stripes-components/lib/MultiColumnList';
+
+const propTypes = {
+  stripes: PropTypes.shape({
+    formatDateTime: PropTypes.func.isRequired,
+    intl: PropTypes.shape({
+      formatMessage: PropTypes.func.isRequired,
+    }),
+  }),
+  failedRenewals: PropTypes.arrayOf(PropTypes.object),
+  successRenewals: PropTypes.arrayOf(PropTypes.object),
+  requestCounts: PropTypes.object,
+  height: PropTypes.number,
+  loanPolicies: PropTypes.object,
+  errorMessages: PropTypes.object,
+};
+
+const defaultProps = {
+  height: 300,
+};
+
+const BulkRenewedLoansList = (props) => {
+  const { formatMessage } = props.stripes.intl;
+  const { stripes, failedRenewals, successRenewals } = props;
+  const iconAlignStyle = { display: 'flex', alignItems: 'center' };
+  const pointerStyle = { cursor: 'pointer' };
+
+  const visibleColumns = [
+    'renewalStatus',
+    'title',
+    'itemStatus',
+    'currentDueDate',
+    'requestQueue',
+    'barcode',
+    'callNumber',
+    'loanPolicy',
+  ];
+
+  return (
+    <MultiColumnList
+      interactive={false}
+      height={props.height}
+      contentData={[...failedRenewals, ...successRenewals]}
+      visibleColumns={visibleColumns}
+      columnMapping={{
+        renewalStatus: formatMessage({ id: 'ui-users.brd.header.renewalStatus' }),
+        title: formatMessage({ id: 'ui-users.brd.header.title' }),
+        itemStatus: formatMessage({ id: 'ui-users.loans.columns.itemStatus' }),
+        currentDueDate: formatMessage({ id: 'ui-users.loans.columns.dueDate' }),
+        requestQueue: formatMessage({ id: 'ui-users.loans.details.requests' }),
+        barcode: formatMessage({ id: 'ui-users.information.barcode' }),
+        callNumber: formatMessage({ id: 'ui-users.loans.details.callNumber' }),
+        loanPolicy: formatMessage({ id: 'ui-users.loans.details.loanPolicy' }),
+      }}
+      formatter={{
+        renewalStatus: loan => {
+          if (failedRenewals.filter(loanObject => loanObject.id === loan.id).length > 0) {
```

Comment:
If we move the rendering of these Popovers and components into a parent, can we reuse the `LoanList` component? So much of this is similar that it feels off to have something so functionally similar and basically rewrite the whole thing.

Reply:
The only other thing I can see is the ordering of failedRenewals and successRenewals but again, the parent could just do that instead.

Reply:
@doytch Sure, let me look into this

Reply:
@doytch Looks like refactoring would add more complexity than what it is now. Because, in that case, all the properties like `visibleColumns` (the columns are different in both the scenarios), `columnMapping`, `formatter` would need to be sent down as props from the parent. Also, the `LoanList` component has a manifest whereas the `bulkRenewal` doesn't have one (basically a dumb component).  I feel its better to keep it this way as both of them are different and simple enough. Let me know your thoughts.

Reply:
Fair nuff! Thanks for checking! 👍